seqr/views/utils/airtable_utils.py

```diff
@@ -0,0 +1,49 @@
+import requests
+from django.core.exceptions import PermissionDenied
+
+from seqr.utils.logging_utils import SeqrLogger
+from seqr.views.utils.terra_api_utils import is_google_authenticated
+
+from settings import AIRTABLE_API_KEY, AIRTABLE_URL
+
+logger = SeqrLogger(__name__)
+
+class AirtableSession(object):
+
+    RDG_BASE = 'RDG'
+    AIRTABLE_BASES = {
+        RDG_BASE: 'app3Y97xtbbaOopVR'
```

Comment:
because it was never an environment variable, and also we don't support airtable integration for local installs and it never changes so theres no reason to make it one